build/main.go

```diff
@@ -149,6 +154,10 @@ func smartdeps() {
 	cmds := []command{
 		cmd("gx install"),
 		cmd("gx-go rewrite"),
+		// because we call go install on pkgs and that will fail
+		cmd("go get -u go.opencensus.io"),
+		// because go getting opencensus doesn't actually get everything :(
+		cmd("go get -u google.golang.org/grpc"),
```

Comment:
BLOCKING We should not be implementing ad hoc package management for production code. The opencensus code needs to come in through gx like everything else (unless the go modules PR lands soon).

Reply:
agreed, I did this to get the thing working, will remove these on gx add.